### PR TITLE
Update getting-started-windows.md

### PR DESCRIPTION
**Pull request by evilnick**
Update the current getting started for Windows

### CODE REVIEW

**Review by alfsuse — APPROVED** [review]

**Comment by alfsuse** [comment]
I am wondering if we  shouldn't change this page adding more specific content (i.e. use docker on windows + k8s or wsl2 as the section we have for pipelines) /cc @8bitmp3 @RFMVasconcelos 

I can work on it in case.

**Review by alfsuse — APPROVED** [review]

File: content/en/docs/started/workstation/getting-started-windows.md [hunk 1]
```diff
@@ -16,31 +12,44 @@ In addition, with the recent announcement of
 some of the [linux installation options](/docs/started/workstation/getting-started-linux)
```
Comment (RFMVasconcelos) [inline content/en/docs/started/workstation/getting-started-windows.md:12]: There is some documentation on Kubeflow on WSL2, however it's still an overkill, not providing easier access. I'd remove this suggestion and when it becomes easier we can drive users in that direction.

File: content/en/docs/started/workstation/getting-started-windows.md [hunk 2]
```diff
@@ -16,31 +12,44 @@ In addition, with the recent announcement of
...
+Follow the instructions on [Kubeflow on MiniKube](/docs/started/workstation/minikube-linux/) to complete this path.
 
-## Multipass Ubuntu
+### Multipass
 
-Windows users can get Kubeflow with [Multipass](https://multipass.run/#install) 
-by following the instructions on the 
-[Multipass and MicroK8s getting started](/docs/started/workstation/getting-started-multipass/)
-page.
+[Multipass](https://multipass.run/) creates a Linux virtual machine on Windows, Mac or Linux systems. The VM contains a complete Ubuntu operating
```
Comment (RFMVasconcelos) [inline content/en/docs/started/workstation/getting-started-windows.md:50]: Mac -> macOS

**Comment by RFMVasconcelos** [comment]
Thank you for restructuring this @evilnick !

@alfsuse, thank you for your offer! :)
For the moment I think the restructuring that @evilnick did is already a great step forward as this doc has been outdated and messy for quite a while. I propose we close this PR to have these changes take effect and then potentially evaluate adding more complexity in other PRs.

**Pull request merged**